code/game/objects/items/toys.dm

```diff
@@ -321,6 +322,17 @@
 	icon_state = "comicstormman"
 	item_state = "comicstormman"
 
+/obj/item/toy/comic/outlandish_tales
+	name = "outlandish tales magazine"
```

Comment:
```suggestion
	name = "\improper Outlandish Tales magazine"
```
if "outlandish tales" is a trademark or brand name, otherwise ignore this comment.